src/gateways/postgres/plugin.js

```diff
@@ -17,18 +17,18 @@ export default class PostgresGateway extends Gateway {
     });
   }
 
-  async write(datum) {
+  async write(document) {
     const client = await this.pool.connect();
 
     try {
       await client.query("BEGIN");
 
-      const { product_id, style_id, size, quantity } = datum;
+      const { product_id, style_id, size, quantity } = document;
       const queryText =
         "INSERT INTO products (product_id, style_id, size, quantity) VALUES ($1, $2, $3, $4) RETURNING product_id";
       const response = await client.query(queryText, [
-        JSON.stringify(`${product_id}`),
-        JSON.stringify(`${style_id}`),
+        `${product_id}`,
+        `${style_id}`,
         size,
```

Comment:
**Annotation:** Unnecessary JSON.stringify method call here. Backticks automatically converts the item into a string.